src/DonationForms/resources/app/fields/SectionNode.tsx

```diff
@@ -47,12 +47,16 @@ function SectionNode({node}: {node: Node}) {
 
     if (isField(node) && fieldTemplateExists(node)) {
         if (node.type === 'gateways') {
+            // @ts-ignore
```

Comment:
I'm seeing alot of these `@ts-ignore` additions that is concerning.  These should _only_ be used as a last resort - and considering they were not needed previously tells me there is some kind of internal problem we should be fixing up the chain.

can we remove them?

Reply:
After installing version `1.0.0` of the new package, the IDE started to highlight all places that use the node attribute and it also started preventing the `npm run dev` from finishing the compile process due this error:

```
TS2322: Type { node: Field; } is not assignable to type IntrinsicAttributes Property node does not exist on type IntrinsicAttributes
``` 

So now, after updating the package for the `1.1.2` version, I tried to remove these `@ts-ignore` tags but the problem keeps happening even with the new package as you can check in the attached screenshot. 

![image](https://github.com/impress-org/givewp/assets/16308864/b54c4e6b-8de7-4d8e-8869-ef95d7f385e2)

Reply:
@jonwaldstein One more thing, if we execute the `npm run dev:v3` the problem doesn't happen. So, may this is happening due to this: 

> ...running npm run dev on GiveWP runs both the Laravel Mix dev and WP Scripts dev. Laravel Mix has strict TypeScript checking in place, while WP Scripts does not. It's annoying. Usually, the best route is to simply address the TS warnings/errors, using `@ts-ignore` when needed.

**Source:** https://github.com/impress-org/form-builder-library/pull/1#issuecomment-1836378810

Any thoughts?

Reply:
Okay, i'd like to explore the problem more.  Although Laravel mix points out typescript errors while wp-scripts does not - that alone does not warrant adding `@ts-ignore` as it defeats the purpose of using typescript all together.  

I would only recommend using `@ts-ignore` when resolving the TS error is _not_ possible - say from an external library.  However, if typescript is pointing out an error in our code than we should be resolving that error.

I do understand certain dependencies can add conflicting typescript errors but those should be _addressed_ - not be _ignored_, so we can figure out _why_ for the long-term.

Reply:
resolved [c471a2a](https://github.com/impress-org/givewp/pull/7122/commits/c471a2a892497d004799daf21b90bbd2ae0d9474)